packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -179,12 +179,14 @@ export class LoginPage {
 				await Promise.all( [
 					// Shorter than usual timoout, because with a cookie file the login process
 					// should not take more than a few seconds.
-					this.page.waitForNavigation( { url: landingUrl, waitUntil: 'load', timeout: 15 * 1000 } ),
+					this.page.waitForNavigation( { url: landingUrl, waitUntil: 'load', timeout: 7 * 1000 } ),
 					this.page.goto( getCalypsoURL( '/' ) ),
 				] );
 				return;
-			} catch {
-				console.log( 'Unable to log in using cookie file, retrying a normal login.' );
+			} catch ( err: unknown ) {
+				if ( err instanceof Error ) {
+					console.log( `${ err.message }, retrying username/password based authentication.` );
```

Comment:
Maybe this log isn't necessary, but I have included it so users know what's happening.